pypesto/C.py

```diff
@@ -84,12 +84,13 @@ class EnsembleType(Enum):
 
 
 ###############################################################################
-# HIERARCHICAL
+# HIERARCHICAL SCALING + OFFSET
```

Comment:
please remove me from CODEOWNERS

Reply:
owner via https://github.com/orgs/ICB-DCM/teams/pypesto-maintainers, because there is no explicit owner for that file (which makes sense, I guess)

Reply:
for `C.py` I agree its fine, but for the other files it's the same issue and I don't want a review request for any PR that changes something in the documentation ...